docs/deploy.md

```diff
@@ -162,10 +165,11 @@ Every deployment will need an [App](https://developer.github.com/apps/).
 #### Repository Permissions
 
 - Administration: **Read & Write**
-- Contents: **Read only**
+- Checks: **Read & Write**
+- Commit statuses: **Read & Write**
+- Contents: **Read & Write**
```

Comment:
Do you know why this is needed? Does this have to do with `auto_init` stuffs?

Reply:
@moensch The `Contents` is needed to be able to rename the default branch based on what are in the settings, the `Checks` is for the `nop` check on the PR I think,  and not entirely sure on `Commit statuses`  - I think having to deal with the [checks](https://docs.github.com/en/rest/reference/commits#list-commit-statuses-for-a-reference) again